tests/nobackground_pytest.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Testing Sphinx Writer
+
+    Test the ability to write Gherkin data to a SphinxWriter object,
+    with a multiline feature description for pytest.
+
+    This includes several paragraphs of description,
+    to demonstrate how blank lines are handled
+    in the pytest-bdd parser.
```

Comment:
The feature file name mentions that this doesn't have a background, but maybe a decent idea to mention it here as well since that is an explicit thing we are testing. Also, is there a test for the behave parser without a background? If not, would be good to add it.